pwndbg/commands/context.py

```diff
@@ -49,19 +51,19 @@ def context(*args):
     Accepts subcommands 'reg', 'code', 'stack', 'backtrace', and 'args'.
     """
     if len(args) == 0:
-        args = ['reg','code','stack','backtrace','args']
+        args = str(config_context_sections).split()
 
     args = [a[0] for a in args]
 
     result = []
 
     result.append(M.legend())
-    if 'r' in args: result.extend(context_regs())
-    if 'c' in args: result.extend(context_code())
-    if 'c' in args: result.extend(context_source())
-    if 'a' in args: result.extend(context_args())
-    if 's' in args: result.extend(context_stack())
-    if 'b' in args: result.extend(context_backtrace())
+    for arg in args:
+        if 'r' == arg: result.extend(context_regs())
+        if 'd' == arg: result.extend(context_code())
+        if 's' == arg: result.extend(context_source())
+        if 's' == arg: result.extend(context_stack())
```

Comment:
The `'s'` is doubled. If that is intentional, go with `result.extend((context_source(), context_stack())`.

I would also go with `if` + `elif`s instead of multiple `if`. If the arg is always one of `r,d,s,b` we can also make a dict mapping here.